google-cloud-bigquery/lib/google/cloud/bigquery/project.rb

```diff
@@ -1378,6 +1378,20 @@ def external url, format: nil
         #   service. Calls made on this object will raise errors if the resource
         #   does not exist. Default is `false`. Optional.
         # @param [String] project_id The GCP Project where the dataset lives.
+        # @param [Integer] access_policy_version Optional. The version of the
+        #   provided access policy schema. Valid values are 0, 1, and 3.
+        #   Requests specifying an invalid value will be rejected. This
+        #   version refers to the schema version of the access policy and not
+        #   the version of access policy. This field's value can be equal or
+        #   more than the access policy schema provided in the request. For
+        #   example, requests with conditional access policy binding in datasets
+        #   must specify version 3. But dataset with no conditional role
+        #   bindings in access policy may specify any valid value or leave the
+        #   field unset. If unset or if 0 or 1 value is used for dataset with
+        #   conditional bindings, request will be rejected. This field will be
+        #   mapped to [IAM Policy version]
+        #   (https://cloud.google.com/iam/docs/policies#versions) and will
```

Comment:
Don't allow a line break (or any whitespace) between the text and the URL in a markdown link. Markdown will not be able to recognize the syntax, and will not render it correctly.